HousingSearchApi/V1/Controllers/GetPersonListController.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Threading.Tasks;
+using HousingSearchApi.V1.Boundary.Responses.Metadata;
+using HousingSearchApi.V1.Domain;
+using HousingSearchApi.V1.UseCase.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+
+namespace HousingSearchApi.V1.Controllers
+{
+    [ApiVersion("2")]
+    [Produces("application/json")]
+    [Route("api/v1/search/persons")]
+    public class GetPersonListController : BaseController
+    {
+        private readonly IGetPersonListUseCase _getPersonListUseCase;
+
+        public GetPersonListController(IGetPersonListUseCase getPersonListUseCase)
+        {
+            _getPersonListUseCase = getPersonListUseCase;
+        }
+
+        [ProducesResponseType(typeof(APIResponse<GetPersonListResponse>), 200)]
+        [ProducesResponseType(typeof(APIResponse<BadRequestException>), 400)]
```

Comment:
Should this not be a 404 response as agreed on Swagger

Reply:
I don't think so @humulla. 404 is for not found, where this is a 400 for a bad request where (I am guessing here) the input search parameters are invalid.

Reply:
I can add a 404 next ticket, np